### PR TITLE
scylla load test

### PR DESCRIPTION
**Pull request by larisau**
not ready for merge, can be modified

### CODE REVIEW

File: load_test.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    """
+    Test a Scylla cluster load over a time period.
+
+    :avocado: enable
+    """
+
+    def test_load(self):
+        """
+        Run cassandra-stress with params defined in data_dir/scylla.yaml
```
Comment (amoskong) [inline load_test.py]: The yaml file is configurable.
 in data_dir/scylla.yaml -> in yaml config

Comment (amoskong) [reply load_test.py]: Can you describe the content and purpose of this test here? it looks running heavy load (8 loaders) for 1 db node, and persist 7 days.

Comment (amoskong) [reply load_test.py]: Can we reuse longevity_test with a new config?

Comment (larisau) [reply load_test.py]: I don't think so, this test is without nemesis

Comment (amoskong) [reply load_test.py]: We can make nemesis configurable in longevity_test. If there is a big gap between new loader test and longevity_test. We absolutely need a new file.

Comment (larisau) [reply load_test.py]: Load test intended to catch up scylla timeouts/exceptions, currently it does not contain the check itself, and I'm not sure we want to mix it with longevity.

Comment (amoskong) [reply load_test.py]: Ok

**Comment by roydahan** [comment]
@larisau , what is missing?
please include this test in the SCT_cases job in Jenkins when it's ready.

**Comment by roydahan** [comment]
The test still missing the "verification of expected result part".
We are still unable to run this test, so we don't know exactly what to look for, right? (@larisau )

**Pull request closed without merging**